index.html

```diff
@@ -54,13 +54,13 @@
         </div>
         
         <!--Add items bar-->
-        <div class="container-fluid" id="listMembersBar">
+        <div class="add-items-bar container-fluid">
             <div class="col-xs-9">
                 <input class="form-control input-sm" id="text-input" type="text"/>
             </div>
             <!--Add button-->
             <div class="col-xs-3">
-                <button id="add-button" type="button" class="btn btn-primary btn-xs" onclick="addItem()">Add</button>
```

Comment:
Well, I guess it was being used in CSS, but it's usually better to use classes instead of ID's